.github/workflows/ci.yml

```diff
@@ -101,44 +101,46 @@ jobs:
           files: ./coverage.out
           verbose: true
 
+  go-integration-setup:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/alicenet-config
+      - name: npm Compile
+        working-directory: ./bridge
+        shell: bash
+        run: npm run compile
+      - uses: actions/upload-artifact@v3
+        with:
+          name: hardhat-files
+          path: bridge/artifacts/
+      - id: set-matrix
+        working-directory: layer1/executor/
+        run: |
+          targets=($(grep -R 'Test' ./ | awk -F 'func' '{ print $2 }' | awk -F '(' '{ print $1 }'))
+          printf -v joined '"%s",' "${targets[@]}"
+          echo "::set-output name=matrix::[${joined%,}]"
```

Comment:
120+ checks for CI is a _little_ much. Even at the highest non-enterprise plan we only get 60 executors, which means we'd saturate the full set twice over for each CI job.